### ISSUE TITLE
Suppression for SE_NO_SERIALVERSIONID on Groovy files

### ISSUE THREAD

**Issue by markelliot** [palantir/baseline#1]
After upgradling to 3.0, one of my Gradle plugin projects started to fail on findbugs errors finding Closures defined in my Groovy code did not also include a serialVersionUID (evidently because something in the upgrade stack has caused Closure to become recognized as Serializable).

It's not obvious that we want to rewrite our Groovy code to add serialVersionUID to all our Closures, and rather than lose all Findbugs support, I propose we add the following filter:

    <Match>
        <Source name="~.*\.groovy" />
        <And>
            <Bug pattern="SE_NO_SERIALVERSIONID"/> 
        </And>
    </Match>

**Comment by uschi2000** [comment]
Yeah, that's reasonable. Also wouldn't be surprised if FindBugs fixed this at some point?
